include/lbann/data_readers/data_reader_jag_conduit.hpp

```diff
@@ -104,6 +104,8 @@ class data_reader_jag_conduit : public generic_data_reader {
   ~data_reader_jag_conduit() override;
   data_reader_jag_conduit* copy() const override { return new data_reader_jag_conduit(*this); }
 
+  void setup(int num_io_threads) override;
```

Comment:
Doesn't override. Missing the `thread_pool` parameter.

Reply:
Fixed.